PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -3394,7 +3394,15 @@ def _replace_paths_in_code(self, co):
 
         code_func = type(co)
 
-        if hasattr(co, 'co_kwonlyargcount'):
+        if hasattr(co, 'co_posonlyargcount'):  # >= 3.8
+            return code_func(
+                co.co_argcount, co.co_posonlyargcount,
+                co.co_kwonlyargcount, co.co_nlocals,
+                co.co_stacksize, co.co_flags, co.co_code,
+                tuple(consts), co.co_names, co.co_varnames,
+                new_filename, co.co_name, co.co_firstlineno,
+                co.co_lnotab, co.co_freevars, co.co_cellvars)
+        elif hasattr(co, 'co_kwonlyargcount'):
```

Comment:
Thanks for these modulegraph changes - leave them for now, though you'll have to remove them later once I port them onto my provate modulegraph move branch. The new modulegraph is radically different and I'm keeping the work local for now.

Reply:
Then we can just merge these changes in, and leave them until modulegraph is replaced with moduelgraph2, right?